pkg/validations/validate.go

```diff
@@ -29,7 +29,7 @@ type rule struct {
 // validationKwargs represent the optional keyword arguments and their values in a validation annotation.
 type validationKwargs struct {
 	when         *starlark.Callable
-	whenNullSkip bool
+	whenNullSkip *bool
```

Comment:
```suggestion
	whenNullSkip *bool  // default: true if value is nullable
```

Explicitly define the meaning of `nil`?